### PR TITLE
looper-> master

### PR DESCRIPTION
**Pull request by GilStein1**
a wrapper for a for loop

### CODE REVIEW

**Review by aaa-png — DISMISSED** [review]

File: src/main/java/frc/utils/GBLooper.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package frc.utils;
+
+import java.util.function.Consumer;
+
+public class GBLooper {
```
Comment (YHerm) [inline src/main/java/frc/utils/GBLooper.java:5]: why GB?

Comment (YHerm) [inline src/main/java/frc/utils/GBLooper.java:5]: call it a util name and not object

File: src/main/java/frc/utils/LoopingUtils.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import java.util.function.Consumer;
+
+public class LoopingUtils {
+
+	public static void loop(Consumer<Integer> loop, int iterations) {
+		for (int index = 0; index < iterations; index++) {
+			loop.accept(index);
+		}
+	}
+
```
Comment (NoamRosenberg08) [inline src/main/java/frc/utils/LoopingUtils.java:12]: loop (Consumer<Integer>... loops )

**Review by NoamRosenberg08 — APPROVED** [review]

File: src/main/java/frc/utils/LoopingUtils.java (hunk 2 above)
Comment (NoamRosenberg08) [reply src/main/java/frc/utils/LoopingUtils.java:12]: nvm

**Pull request closed without merging**